org.syphon.Syphon.metainfo.xml

```diff
@@ -16,13 +17,9 @@
   <developer_name>Syphon Organization, LLC</developer_name>
   <description>
     <p>Syphon aims to be built on the foundations of privacy, branding, and user experience in an effort to pull others away from proprietary chat clients to the matrix protocol.</p>
-
-    <p>Website: https://syphon.org</p>
-    <p>Source Code: https://github.com/syphon-org/syphon</p>
-    <p>Matrix Space: https://matrix.to/#/#syphon-space:matrix.org</p>
```

Comment:
I would like to keep some kind of reference to the Matrix space.

Reply:
It's not possible to add plain URLs to the descriptions, so I removed https part.